docs/application/web/api/5.0/device_api/wearable/tizen/alarm.html

```diff
@@ -838,7 +839,7 @@ <h4>Methods</h4>
           </div>
```

Comment:
Regarding deprecation policy https://docs.tizen.org/application/web/tutorials/deprecation-policy, API deprecated in 4.0 is still valid on 5.0 and become unavailable since next version 5.5.
Because of this we left mentioned comment in 5.0 documentation, but it is removed in 5.5.